x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client.ts

```diff
@@ -395,3 +407,89 @@ const conditions = {
     return [this.isFullyTranslated(), this.isNotInstalled()];
   },
 };
+
+const missing = (direction: estypes.SortOrder = 'asc') =>
+  direction === 'desc' ? '_last' : '_first';
+
+const sortingOptions = {
+  author(direction: estypes.SortOrder = 'asc'): estypes.SortCombinations {
```

Comment:
Calling this "author" is something specific to the UI, In the API we should call it _matchedPrebuiltRule_ or _customTranslation_ (opposite).

Reply:
done